tests/playwright/src/specs/image-smoke.spec.ts

```diff
@@ -51,6 +51,7 @@ test.beforeAll(async () => {
 });
 
 test.afterAll(async () => {
+  // test
```

Comment:
Does this comment need to be here? Maybe it's an artifact from some previous debugging?